ena-submission/environment.yml

```
@@ -22,6 +22,7 @@ dependencies:
   - xmltodict
   - biopython
   - pytz
+  - ruff =0.13.1
```

ok - lets add a comment here to edit the .pre-commit-config.yaml if we update it
```suggestion
  - ruff =0.13.1. # if changing update .pre-commit-config.yaml as well
```